test/native/integration-test-helpers/rich-text-select.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Internal dependencies
+ */
+import { typeInRichText } from './rich-text-type';
+
+/**
+ * Select text of a RichText component.
+ *
+ * @param {import('react-test-renderer').ReactTestInstance} richText       RichText test instance.
+ * @param {number}                                          selectionStart Selection start position.
+ * @param {number}                                          selectionEnd   Selection end position.
+ *
+ */
+export const selectTextInRichText = (
+	richText,
+	selectionStart = 0,
```

Comment:
As mentioned in [this comment](https://github.com/WordPress/gutenberg/pull/49804/files#r1165601733), probably we could make this argument required.